src/hotspot/share/runtime/thread.cpp

```diff
@@ -1662,87 +1661,96 @@ bool JavaThread::resize_all_jvmci_counters(int new_size) {
 
 // A JavaThread is a normal Java thread
 
-void JavaThread::initialize() {
+JavaThread::JavaThread() :
   // Initialize fields
 
-  set_saved_exception_pc(NULL);
-  _anchor.clear();
-  set_entry_point(NULL);
+  _on_thread_list(false),
+  DEBUG_ONLY(_java_call_counter(0) COMMA)
+  _entry_point(nullptr),
```

Comment:
I didn't realize we had made the NULL -> nullptr switch yet ??

Reply:
Kim said I should use it:
Kim Barrett  10:12 AM
Quoting the style guide:
nullptr
Prefer nullptr (n2431) to NULL. Don't use (constexpr or literal) 0 for pointers.
For historical reasons there are widespread uses of both NULL and of integer 0 as a pointer value.
I don't know how many people actually read the changes I made.